Client/Frontend/Strings.swift

```diff
@@ -179,11 +179,11 @@ extension String {
     /// Identifiers of all new strings should begin with `FirefoxHome.`
     public struct FirefoxHomepage {
         public struct Common {
-            public static let SitesCount = MZLocalizedString(
-                "FirefoxHomepage.Common.SitesCount.v101",
+            public static let PagesCount = MZLocalizedString(
+                "FirefoxHomepage.Common.PagesCount.v101",
```

Comment:
Had a discussion with @dnarcese, to give enough time for translations we can merge this change in v112. We can make sure the key reflects that with: 
```suggestion
            public static let PagesCount = MZLocalizedString(
                "FirefoxHomepage.Common.PagesCount.v112",
```

The PR title can also be adapted. We are cutting off the v111 release today so we'll be able to merge this tomorrow.